Source/SvgDocument.cs

```diff
@@ -74,6 +76,33 @@ private static int GetSystemDpi()
             }
             return _fontDefns;
         }
+        internal FontFamily[] PrivateFontDefns()
+        {
+            if (_privateFonts == null)
+            {
+                _privateFonts = new PrivateFontCollection();

```

Comment:
There is [remarks](https://docs.microsoft.com/dotnet/api/system.drawing.text#remarks) in the API reference.
What do you think about this ?